src/routes/quiz/biography/+page.svelte

```diff
@@ -64,7 +87,7 @@
 					{/if}
 				</p>
 			</section>
```

Comment:
Use the following syntax to improve readability (optional)
```suggestion
				class:bg-error-200={!!form && form?.correct}
```